knexfile.js

```diff
@@ -0,0 +1,8 @@
+require('babel-register')();
```

Comment:
The `knexfile.js` is auto-loaded when the migrations cli is run, which is a separate entry-point. I'll add a comment to explain this